webserver/views/api/v1/core.py

```diff
@@ -267,6 +267,93 @@ def get_many_highlevel():
+    raised stating the missing features message.
+    
+    Returns a string of feature paths with alias:
+        "llj.data->'feature1' AS feature1, ..., llj.data->'featureN' AS featureN"
+    """
+    features_param = request.args.get("features")
+    if not features_param:
+        raise webserver.views.api.exceptions.APIBadRequest("Missing `features` parameter")
+
+    selectable_features = ['lowlevel.average_loudness', 
+                            'lowlevel.dynamic_complexity', 
+                            'metadata.audio_properties.replay_gain', 
+                            'rhythm.beats_count', 
+                            'rhythm.beats_loudness.mean', 
+                            'rhythm.bpm', 
+                            'rhythm.bpm_histogram_first_peak_bpm.mean', 
+                            'rhythm.bpm_histogram_second_peak_bpm.mean', 
+                            'rhythm.danceability', 
+                            'rhythm.onset_rate', 
+                            'tonal.chords_key', 
+                            'tonal.chords_scale', 
+                            'tonal.key_key', 
+                            'tonal.key_scale', 
+                            'tonal.tuning_frequency', 
+                            'tonal.tuning_equal_tempered_deviation']
+
+    parsed_features = []
+    raw_paths = []
+    for feature in features_param.split(';'):
+        if feature in selectable_features:
+            raw_paths.append(feature)
+            # Build feature path
+            feature_path = 'llj.data'
+            for element in feature.split('.'):
+                feature_path += '->\'' + element + '\''
+            parsed_features.append(feature_path)
+
+    # Remove duplicates, preserving order
+    seen = set()
+    parsed_features = [x for x in parsed_features if not (x in seen or seen.add(x))]
+
+    features_string = parsed_features[0] + ' AS "' + raw_paths[0] + '", '
+    for feature, alias in zip(parsed_features[1:len(parsed_features)-1], raw_paths[1:len(raw_paths)-1]):
+        features_string += feature + ' AS "' +  alias + '", '
+    features_string += parsed_features[len(parsed_features)-1] + ' AS "' + raw_paths[len(raw_paths)-1] + '"'
+
+    return features_string
+
+
+@bp_core.route("/low-level/select", methods=["GET"])
+@crossdomain()
+def get_many_select_features():
+    """Get a specified subset of low-level data for many recordings at once.
+    
+    **Example response**:
+
+    .. sourcecode:: json
+       {"mbid1": {"offset1": {document},
+                  "offset2": {document}},
+        "mbid2": {"offset1": {document}}
+       }
+    
+    :query recording_ids: *Required.* A list of recording MBIDs to retrieve.
+        
+        Takes the form `mbid[:offset];mbid[:offset]`. Offsets are optional, and should
+        be >= 0
+    
+    :query features: *Required.* A list of features to be returned for each mbid.
+        
+        Takes the form `feature1;feature2`.
```

Comment:
we should list the features here. One way you could do this is like I did for the max number of bulk queries: 
https://github.com/metabrainz/acousticbrainz-server/blob/cc359f8ea5c580fbdb0e0a1c0bb71004ca4f380c/webserver/views/api/v1/core.py#L260